esmvaltool/recipes/recipe_modes_of_variability_wp4.yml

```
@@ -0,0 +1,64 @@
+# ESMValTool
+# recipe_modes_of_variability_wp4.yml
+---
+documentation:
+  description: |
+    Tool to compute the RMSE between the observed and modelled patterns of
+    variability obtained through classification and their relative relative
+    bias (percentage) in the frequency of occurrence and the persistence of
+    each mode.
+
+  authors:
+    - torr_ve
+    - fuck_ne
+    - cort_ni
+    - guem_vi
+    - hunt_al
+    - manu_ni
+
+  projects:
+    - c3s-magic
+
+  references:
+    - fuckar
+
+datasets:
+  - {dataset: MIROC5, project: CMIP5,  start_year: 1961,  end_year: 1990, ensemble: i0p1, exp: historical, type: exp}
+  - {dataset: MIROC5, project: CMIP5,  start_year: 2006,  end_year: 2026, ensemble: r1i1p1, exp: rcp85, type: exp}
+
+preprocessors:
+  preproc:
+    regrid:
+      target_grid: MIROC5
+      scheme: linear
+    mask_fillvalues:
+      threshold_fraction: 0.95
+    extract_region:
+      start_longitude: -80
+      end_longitude: 50
+      start_latitude: 20
+      end_latitude: 80
+
+diagnostics:
+  weather_regime:
+    description: Compute modes of variability.
+    variables:
+      psl:
+        preprocessor: preproc
+        mip: day
+        field: T2M
```

```suggestion
        field: T2D
```
this is daily data, not monthly